src/models/transaction.ts

```diff
@@ -2,12 +2,12 @@ export class Transaction {
   chain: string
   txHash: string
   blockNumber: number
-  blockTimestamp: number
+  blockDate: string
```

Comment:
if we're getting a Date object back from ethers can we keep using a Date type throughout our code?  That's a stronger type match than just using string.

Reply:
Sadly, the cbor encoding doesn't support it. Based on your other comments, I think the best thing is just to use number and millis since epoch.

Reply:
I meant hold on to a Date reference in our code, then serialize it to a number before writing it out to IPFS

Reply:
Good suggestion, will look into changing it there.